roles/horizon/tasks/main.yml

```diff
@@ -203,26 +124,79 @@
+              dest=/etc/httpd/conf.d/openstack-dashboard.conf
+    notify:
+      - reload apache
+
+  - name: link local settings to /etc/openstack-dashboard
+    file: src=/etc/openstack-dashboard/local_settings.py
+          dest=/usr/share/openstack-dashboard/openstack_dashboard/local/local_settings.py
+          owner=root
+          group="{{ apache_user }}"
+          mode=0644
+          state=link
   when: openstack_install_method == 'distro'
 
-- name: Enable/Disable lbaas dashboard (ubuntu)
-  file:
-    src: "{{ horizon.horizon_lib_dir }}/lib/python2.7/site-packages/neutron_lbaas_dashboard/enabled/_1481_project_ng_loadbalancersv2_panel.py"
-    path: "{{ horizon.horizon_lib_dir }}/lib/python2.7/site-packages/openstack_dashboard/local/enabled/_1481_project_ng_loadbalancersv2_panel.py"
-    state: "{{ (neutron.lbaas.enabled | bool) | ternary('link', 'absent') }}"
-  notify: reload apache
-  when: os == 'ubuntu'
+- name: openstack dashboard config (centos)
+  template: src=etc/apache2/sites-available/openstack_dashboard.conf
+            dest=/etc/httpd/conf.d/openstack_dashboard.conf
+  when:
+    - os == 'rhel'
+    - openstack_install_method == 'source'
+  notify:
+    - reload apache
+# Source horizon config
+- block:
+  - name: add local_settings to venv
+    file: src=/etc/openstack-dashboard/local_settings.py
+          dest=/opt/openstack/current/horizon/lib/python2.7/site-packages/openstack_dashboard/local/local_settings.py
+          owner=root
+          group="{{ apache_user }}"
+          mode=0644
+          state=link
+        
+  - name: gather static assets from openstack install (source)
+    environment:
+      DJANGO_SETTINGS_MODULE: 'openstack_dashboard.settings'
+      PYTHONPATH: "$PYTHONPATH"
+    shell: "{{ item }}"
+    with_items:
+      - "{{ horizon.horizon_lib_dir }}/bin/django-admin.py collectstatic --noinput"
+      - "{{ horizon.horizon_lib_dir }}/bin/django-admin.py compress --force"
 
-- name: gather static assets from openstack install (ubuntu)
-  environment:
-    DJANGO_SETTINGS_MODULE: 'openstack_dashboard.settings'
-    PYTHONPATH: "$PYTHONPATH"
-  shell: "{{ item }}"
-  with_items:
-    - "{{ horizon.horizon_lib_dir }}/bin/django-admin.py collectstatic --noinput"
-    - "{{ horizon.horizon_lib_dir }}/bin/django-admin.py compress --force"
-  when: os == 'ubuntu'
+  when: openstack_install_method == 'source'
+
+- name: set horizon keystone policy fact
+  set_fact: policy_for_horizon=True
+  when: horizon.customize|default('False')|bool
 
+# Use custom policy to allow cloud_admin admin panel access
+- name: configure custom keystone policy
+  template:
+    src: "roles/keystone/templates/etc/keystone/policy.json"
+    dest: "/etc/openstack-dashboard/keystone_policy.json"
+    mode: 0640
+    owner: "{{ apache_user }}"
+    group: "{{ apache_group }}"
+  when: horizon.customize|default('False')|bool
+  notify:
+    - reload apache
+
+- name: custom horizon logo
```

Comment:
can we just use ternary operator here for dest path s.t one task lay's down logo for distro and non distro?